src/includes/getting-started-config/javascript.mdx

```diff
@@ -1,15 +1,14 @@
-You should `init` the Sentry Browser SDK as soon as possible during your page load:
+This snippet includes:
```

Comment:
imo you do not need to tell developers this. if we have to spell out everything we're doing we're going to end up with too much boilerplate. i recognize the goal but I think we need to find a simplified way to achieve "want more information on releases? go here but you can figure out how to at least configure it without that"

Reply:
Trimmed it down further.